api/src/main/scala/hmda/api/processing/lar/SingleLarValidation.scala

```diff
@@ -28,6 +29,9 @@ class SingleLarValidation extends Actor with ActorLogging with LarEngine {
     case CheckValidity(lar) =>
       log.debug(s"Checking validity on LAR: ${lar.toCSV}")
       sender() ! validationErrors(lar, checkValidity)
+    case CheckQuality(lar) =>
+      log.debug(s"Checking quality on LAR: ${lar.toCSV}")
+      sender() ! validationErrors(lar, checkQuality)
```

Comment:
I can replace this line with `sender() ! Nil` and all the tests still pass. I'd prefer if we cover that case. Not necessarily with an integration test, but with something.